packages/lesswrong/lib/publicSettings.ts

```diff
@@ -82,3 +82,7 @@ export const elicitSourceURL = new DatabasePublicSetting('elicitSourceURL', 'htt
 export const elicitSourceId = new DatabasePublicSetting('elicitSourceId', 'XCjOpumu-')
 
 export const mapboxAPIKeySetting = new DatabasePublicSetting<string | null>('mapbox.apiKey', null) // API Key for the mapbox map and tile requests
+
+export const mailchimpAPIKeySetting = new DatabasePublicSetting<string | null>('mailchimp.apiKey', null)
```

Comment:
Love that you've figured out how to group settings. Make sure you push the credentials to prod before deploying.